### PR TITLE
Partial CSS-Rework code

### PR DESCRIPTION
**Pull request by Flyken271**
This is the partial code for 'Project-Css-Rework', this PR is intended for preliminary review by repomaster.

### CODE REVIEW

File: templates/add-movie.html [hunk 1]
```diff
@@ -6,15 +6,15 @@
     <div id="addMovieForm">
 		{{if .FormfillEnabled}}
         <div class="movieInput">
-            <div{{if .ErrTitle}} class="errorMessage"{{end}}><label for="MovieName">Movie Title</label></div>
-			<div><textarea name="MovieName" id="MovieName" style="width:400px">{{if .ValTitle}} {{.ValTitle}}{{end}}</textarea></div>
+            <div{{if .ErrTitle}} class="errorMessage"{{end}}><label class="MovieNameLabel">Movie Name</label></div>
```
Comment (CptPie) [inline templates/add-movie.html:9]: Just a small thing, but i would prefer that field being called "Movie Title" instead of "Movie Name" (imo Title is the more descriptive word)

File: templates/base.html [hunk 2]
```diff
@@ -10,7 +10,13 @@
     </head>
     <body>
         <div id="header">
-            <div id="headTitle"><a href="/" class="titleLink">MoviePolls</a>{{if .PageTitle}} - {{.PageTitle}}{{end}}</div>
```
Comment (CptPie) [inline templates/base.html:13]: Why did you remove the PageTitle here?

File: templates/add-movie.html (hunk 1 above)
Comment (Flyken271) [reply templates/add-movie.html:9]: this was not done by me, i kept all existing names the same, even if i added anything to the name, the original is still in there

File: templates/base.html (hunk 2 above)
Comment (Flyken271) [reply templates/base.html:13]: the page title doesn't fit in the new header, will add it back when i know how to add it back into the new header

**Review by CptPie — COMMENTED** [review]
For now it looks fine

**Comment by CptPie** [comment]
So whats going on with this PR - are you waiting on input or smth?

**Comment by Flyken271** [comment]
> So whats going on with this PR - are you waiting on input or smth?

i honestly think we should push it for now, all of the front-facing CSS is done and only the admin panels and such are still the same, ive been having some life stuff popping up and might not be able to get to the other things right now.

if its cool with both you and zorch it'd be safe to push

**Comment by CptPie** [comment]
> > So whats going on with this PR - are you waiting on input or smth?
> 
> i honestly think we should push it for now, all of the front-facing CSS is done and only the admin panels and such are still the same, ive been having some life stuff popping up and might not be able to get to the other things right now.
> 
> if its cool with both you and zorch it'd be safe to push

If thats the case convert the Draft back to a PR and i'll review it later/tomorrow

**Comment by Flyken271** [comment]
its been over a year, can we freaking merge this now?

**Comment by zorchenhimer** [comment]
I was waiting until the other PR was merged because I knew changes would need to be made afterwards.

**Comment by CptPie** [comment]
fixes #38 #8

**Comment by zorchenhimer** [comment]
Looks pretty good, but there's some minor things I noticed.

The search box collides with the message for the last movies watched (this is hard to capture when the box is expanded):
image

The tags can overlap the movie description:
image

When adding a movie, the remarks textarea has a different style:
image

**Comment by Flyken271** [comment]
> Looks pretty good, but there's some minor things I noticed.
> 
> The search box collides with the message for the last movies watched (this is hard to capture when the box is expanded): image
> 
> The tags can overlap the movie description: image
> 
> When adding a movie, the remarks textarea has a different style: image

the title bar was discussed with pie to put it there and like that, the different style on that text area can be fixed with an extra PR as well with the tags position, could we just merge this as is and i'll get to work on those issues in a new PR right away? (just to get this dang thing out of here and merged? :P )

**Comment by zorchenhimer** [comment]
This was accidentally merged with 3c0c72671f2fddd11e24c131ef490a94eeea45c3.  Whoops.  I then proceeded to break it further.

Everything's back on track now though (a force push happened).
